docs/plugins/mailchimp.rst

```diff
@@ -3,4 +3,47 @@
 MailChimp
 #########
 
-.. vale on
\ No newline at end of file
+.. vale on
+This plugin can send contacts to MailChimp cilents based on Contact actions or Point Triggers.
+
+## Authorize
+
+### Get MailChimp API key
+
+1. Create a MailChimp account.
+2. Go to *Account* / *Extras* / *API Keys* and create a new one.
+3. Copy the created API Key.
+
+### Authorize Mautic - MailChimp plugin
+
+1.Fill in with your MailChimp's account **username** 
+2. Add the **API key**
+3. Click on ***Save & Close***  
+
+## Configure the plugin
+
+Navigate to the *Features* tab in the plugin configuration modal box. You should see this note:
+
+> The Contact Field Mapping tab will appear after selecting a segment and will update after changing the selected segment.
+
+![MailChimp Plugin configuration](plugins-mailchimp-configure.png "MailChimp Plugin configuration")
+
+1. Select the Segment.
+
+   If you don't have a segment in [MailChimp][mailchimp] created yet, go to *MailChimp dashboard* / *Segments* / *Create List* and create one.
```

Comment:
This needs to be converted to using the xref linking macro.

If you need to make a link file to add an external URL, please use the command line tool 'make link'. This will generate a reference macro and a link file for you.

For internal linking check the docs on using the [:ref: role](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-ref-role) for linking within the same document, and using [:doc: to link across pages](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-doc-role).